website/description_issues.qmd

```diff
@@ -106,3 +106,29 @@ It is also not recommended to [mention package names](https://cran.r-project.org
 
 ------------------------------------------------------------------------
 
+# Using Authors@R
+
+
+## Problem
+
+You have not properly declared the authors, maintainers and copyright holders in the Authors@R section of the DESCRIPTION file.
+
+## Solution
+
+Add the `Authors@R` field in the DESCRIPTION file. 
+
+
+### Details
+
+You can find more roles in the help documentation by running `?utils::person`. Make sure the provided email address is actively monitored, as it will be the primary point of contact with CRAN for future updates and fixes.
+
+```{r}
+Authors@R: person("Jane", "Smith", email = "chef@cran-cookbook.com",
+  role = c("aut", "cre"))
+```
+
+
```

Comment:
Maybe you can add a paragraph (or callout note) to inform that the `Maintainer` and `Author` fields are auto-generated based on the `Authors@R` and therefore don't need to be added manually. 
Adding them by hand varying from the auto-generated version will cause an automatic rejection on CRAN.